lib/ui/view/dm_user/.vscode/launch.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Kindly remove the vscode commit, you can use it locally on your PC

Reply:
Done